### ISSUE TITLE
Figure outline for paper

### ISSUE THREAD

**Issue by ngreenwald** [angelolab/cell_classification#29]
Disclaimer: Start supplement figures really at the last call

- Figure 1: Dataset overview
  - Summary diagram of dataset construction process. Inputs: segmentation, image data, cell labels. Transform segmentation + cell labels into positive/negative mask for each marker. Combine datasets together. Manual curation of small subset for model evaluation
  - Number of cells in dataset, broken down by tissue types, image platforms
  - Distribution of cells by cell type (Lymphocytes, Monocytes, Other Immune, Epithelial, Stromal, Cancer, Other)
  - Distribution of channels by channel type (Membrane vs. nuclear vs. cytoplasmic vs. ECM protein alternatively lineage vs. phenotypic vs. structural vs. functional marker)
  - Estimate: number of hours to manually annotate test dataset. number of hours to preprocess all training data (total hours to prepare dataset with conversion matrix vs annotating full dataset (based on testset time consumption estimates) vs. only test set, log scale)  OPTION 2 (total number of hours to prepare noisy groundtruth normalized by number of cells vs. time to annotate test dataset normalized by number of cells)

- Figure 2: Model benchmarking
  - Diagram of ProMix workflow: predict on each cell, identify high confidence and correct, discard others. Cartoon version. 
  - Paired image overlay of channel data and model confidence colored by channel
  - Evaluate the accuracy of the noisy labels, a model trained on only noisy labels, and promix model (bar chart, 4 groups -> precision, recall, f1, specificity and three bars per group for noisy label, baseline, promix)
  - Model accuracy broken down by specific marker, specific cell type (summary level with 5 overarching cell types and markers, for appendix split by all markers and all cell types)
  - Accuracy as a function of marker localization (nuclear vs membrane) 
  - Accuracy on high confidence ground truth vs low confidence ground truth
  - Identify cell features which correlate with model accuracy, such as tissue density, marker heterogeneity, marker sparsity, rare markers, signal to noise ratio, cell size, and plot accuracy as a function of these where there is an interesting (or lack of an interesting) relationship
   
- Figure 3: Benefits of model-based positivity estimates
  - Histogram of model probabilities vs normalized expression
  - Biaxial plots of marker co-expression/mutual exclusivity vs normalized expression (sns.heatmaps or scatterplot with marginal probability histogram)
  - Image overlays of cells with bright and dim, but true, staining for different markers. Show integrated expression (high variation in counts) compared to model probabilities (low variation in confidence)
  - Image overlays of staining from different modalities, showing difference in normalized counts across platforms, with same representation in model probabilities. (Intent: Show that after normalization, there's still differences in dynamic range within and between images, but model works also for these cases).

- Figure 4: Cell classification
  - Show heatmap generated with Pixie using model probabilities
  - Show clustering accuracy metrics using our workflow (compare with plot in Pixie paper)
  - Schematic diagram of where this fits into cell classification pipelines in general, modular way
  - Show similar heatmaps generated with other clustering pipelines, all using model probabilities as inputs
  - Show how model can be run/deployed (Workflow figure, compare to Fig. 6)

Supplementary Table 1: Datasets
- Summary data for each dataset included
- Which clustering algorithm was used to generate the original labels, number of images, cells, markers, segmentation, etc

Supplementary Figures
- Accuracy as a function of dataset size

**Comment by ngreenwald** [comment]
Moved to shared folder
